pkg/services/baremetal/host/host.go

```diff
@@ -112,71 +107,24 @@ func SaveHostAndReturn(ctx context.Context, client client.Client, host *infrav1.
 	return res, nil
 }
 
-func (s *Service) getSSHKeysAndUpdateStatus() (osSSHSecret *corev1.Secret, rescueSSHSecret *corev1.Secret, err error) {
-	// Set ssh status if none has been set so far
```

Comment:
I think this change does not fit to the title of the PR (rate limit handling). Why was this code deleted, or is this just a "on the fly" clean up?

Reply:
Very true. That is not related to the PR title. As it is very minor, I would still prefer keeping it in rather than re-running all tests. Thanks for being attentive!